includes/DataObjects/User.php

```diff
@@ -822,8 +825,15 @@ public function oauthCanCreateAccount()
 		}
 	}
 
+	/**
+	 * @return bool
+	 */
 	public function oauthCanCheckUser()
```

Comment:
Hrm, that might be quite a rabit hole though.  There's probably a *lot* of stuff we can make more private in various places.